### PR TITLE
Updates for adding Tumblr back into the footer.

### PR DESCRIPTION
**Pull request by weerd**
#### What's this PR do?
This PR re-adds the DoSomething Tumblr icon link to the global footer.

#### How should this be reviewed?
👁 

#### Any background context you want to provide?
It's back baby!

#### Relevant tickets
Refs https://trello.com/c/UjrGVggb/410-3-as-a-user-i-would-like-to-be-able-to-get-to-the-ds-tumblr-page-from-the-footer

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/paraneue_dosomething.info [hunk 1]
```diff
@@ -28,18 +28,21 @@ settings[header_explore_campaigns_subtext] = 'Any cause, anytime, anywhere.'
 settings[footer_social_facebook_enabled]   = 1
 settings[footer_social_facebook_url]       = https://www.facebook.com/dosomething
 settings[footer_social_facebook_alt]       = dosomething on Facebook
-settings[footer_social_twitter_enabled]    = 1
-settings[footer_social_twitter_url]        = https://twitter.com/dosomething
-settings[footer_social_twitter_alt]        = @dosomething on Twitter
+settings[footer_social_instagram_enabled]  = 1
```
Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/paraneue_dosomething.info:31]: wait why did we change this- oh it got removed from line 40 in the previous version
wait where did twitter go- oh now its down at line 40

DRUPAL!

**Review by itsjoekent — APPROVED** [review]

File: lib/themes/dosomething/paraneue_dosomething/paraneue_dosomething.info (hunk 1 above)
Comment (weerd) [reply lib/themes/dosomething/paraneue_dosomething/paraneue_dosomething.info:31]: Yeah, I alphabetized it 😏

**Comment by DFurnes** [comment]
Post-merge 👍!

**Comment by ngjo** [comment]
[(3) As a user, I would like to be able to get to the DS Tumblr page from the footer](https://trello.com/c/UjrGVggb/410-3-as-a-user-i-would-like-to-be-able-to-get-to-the-ds-tumblr-page-from-the-footer)

**Pull request merged**